gwadoc/doc_en.py

```diff
@@ -1943,7 +1931,7 @@
 ### Relation: domain_region
 
 relations.domain_region.name.en = "Domain Region"
-relations.domain_region.df.en = "A concept which is a geographical / cultural domain pointer of a given concept."
+relations.domain_region.df.en = "a concept which is a geographical / cultural domain pointer of a given concept"
```

Comment:
```suggestion
relations.domain_region.df.en = "a concept which is a geographical/cultural domain pointer of a given concept"
```

Above you have `amount/piece/portion`, so this just keeps the whitespace styling around `/` consistent. Alternatively add spaces to the one above.